packages/react/.storybook/manager.ts

```diff
@@ -0,0 +1,35 @@
+import { addons } from '@storybook/manager-api';
+
+addons.setConfig({
```

Comment:
Don't you think we could try to import values from `defaultTokens` ? ( https://openfun.github.io/cunningham/?path=/docs/getting-started-customization--docs#use-default-design-tokens )

Reply:
Why not : )